packages/combobox/README.md

```diff
@@ -81,7 +81,7 @@ import { Combobox, ComboboxOption } from '@leafygreen-ui/combobox';
 | ------------------------ | --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
 | `children`               | `<ComboboxOption>`, `<ComboboxGroup>`         | Define the Combobox Options by passing children                                                                                                                                                                                                                                                                                                                                                                                                                     |                          |
 | `multiselect`            | `boolean`                                     | Defines whether a user can select multiple options, or only a single option. When using TypeScript, `multiselect` affects the valid values of `initialValue`, `value`, and `onChange`                                                                                                                                                                                                                                                                               | `false`                  |
-| `initialValue`           | `Array<string>`, `string`                     | The initial selection. Must be a string for a single-select, or an array of strings for multiselect. Changing the `initialValue` after initial render will not change the selection.                                                                                                                                                                                                                                                                                |                          |
+| `initialValue`           | `Array<string>`, `string`                     | The `value` of the initial selection. Mapped to the `value` prop of the provided options. (See note on default `value`s) Must be a string for a single-select, or an array of strings for multiselect. Changing the `initialValue` after initial render will not change the selection.                                                                                                                                                                              |                          |
```

Comment:
If I understand this, what you're saying is that `initialValue` and `value` should follow the same format? If thats the case, it's not that clear to me. Maybe you can be more explicit and say they should follow the same format?

Reply:
`initialValue` and/or `value` should match the `value` prop of a `ComboboxOption`

Reply:
I see, that makes it clearer